src/main/java/net/masterthought/cucumber/json/support/Status.java

```diff
@@ -38,8 +38,15 @@ public String getLabel() {
         return String.valueOf(name().charAt(0)).toUpperCase() + name().substring(1).toLowerCase();
     }
 
-    public String toHtmlClass() {
+    public String toHtmlClass(String name) {
         // TODO: should be moved out of this class
-        return "<div class=\"" + this.getName().toLowerCase() + "\">";
+        String uniqueIdentifier = name.trim().toLowerCase().replaceAll(" ", "");
+        return "<div class=\"" + this.getName().toLowerCase() + "\" cuc=\""+uniqueIdentifier+"\">";
```

Comment:
what is going on here? I see one name is passed as parameter second is taken from current instance - tell me your intent